openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -137,12 +142,19 @@ def _imprint(self, tool, data):
 
     def _update_tool_with_data(self, tool, data):
         """Update tool node name and output path based on subset data"""
-        if "subset" not in data:
+
+        subset = data.get("subset")
+        if subset is None:
             return
 
-        original_subset = tool.GetData("openpype.subset")
-        subset = data["subset"]
-        if original_subset != subset:
+        original_data = tool.GetData("openpype")
+
+        # As data contains an extra key called active, add it to
+        # original_data to get a correct comparison
+        if data.get("active"):
+            original_data["active"] = data["active"]
+
+        if original_data != data:
             self._configure_saver_tool(data, tool, subset)
```

Comment:
I think reconfiguring on "any change of data" isnt' the best way forward. That'd mean that technically if just toggle "Render local" to "render on farm" that it'd trigger the reconfigure logic which could technically change the output path, etc. even if for _whatever reason_ the user had it customized. I feel like we should be explicit about what data changed should result in a the output path to be reconfigured, like  a task change or subset change. Right?

Reply:
This note is still relevant I believe @EmberLightVFX ?

Reply:
Hmm - it's a matter of preference I suppose. I know for a fact that a lot of our artists here have been changing the paths a LOT since what they change it too the publisher doesn't really care as long as it can find the output files. :)

Reply:
Should it maybe be a setting then? To allow the publisher to re-generate the save path or not.
I rather not to not end up with 1000 different settings but if it's a feature that's important for yours and maybe other studios I guess it could be a setting.

Reply:
I don't mind it a crazy amount for it to be a dedicated feature. However, I personally would find it odd that if I just deactivated the instance for example that it'd suddenly start setting the path again. To me: asset, variant or subset changes it makes sense and on anything that tweaks the output. But e.g. disabling an optional validator or alike - then let's avoid touching it if we can would for me be the best solution.

@jakubjezek001 @antirotor what's the thought on this?